### PR TITLE
Js linting and inline migration for simple scripts

### PR DESCRIPTION
**Pull request by bbovenzi**
- Moved simple inline js to its own file with linting as per #14115, other more complicated files will be their own PRs (gantt, graph, tree, dag, dags)
- Replaced jQuery some deprecated functions
- changed all js files to snake_case instead of kebab-case for consistency's sake

### CODE REVIEW

File: airflow/www/templates/airflow/dag_code.html [hunk 1]
```diff
@@ -32,23 +32,8 @@
...
-          'color': 'transparent',
-        });
-      }
-    });
-
-    // pygments generates the HTML so set wrap toggle via js
-    if ("{{ wrapped }}" == "True") {
-      toggleWrap();
-    };
+    const isDemoMode = '{{ demo_mode }}' === 'True';
```
Comment (mik-laj) [inline airflow/www/templates/airflow/dag_code.html]: Ideally, we should manage to delete all inline scripts. For this purpose, we can pass this data as the page metadata.
```javascript
function getMetaValue(name) {
  const el = document.querySelector('meta[name="' + name + '"]')
  if (!el) {
    return
  }
  return el.getAttribute("content")
}

global.getMetaValue = getMetaValue
```
```
<meta name="dag-timezone" content="{{ dag.timezone.name }}">
```
```
var dag_timezone = gettMetaValue("dag-timezone")
```

Comment (bbovenzi) [reply airflow/www/templates/airflow/dag_code.html]: Good idea. Updated,

File: airflow/www/templates/airflow/dag_code.html [hunk 2]
```diff
@@ -21,6 +21,14 @@
 
 {% block page_title %}{{ dag.dag_id }} - Code - Airflow{% endblock %}
 
+{% block head %}
+  {{ super() }}
+  <head>
```
Comment (ashb) [inline airflow/www/templates/airflow/dag_code.html]: This will create duplicate `<head>` tags won't it?

File: airflow/www/static/js/circles.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        .attr('cy', (d) => scly(d.y))
+        .attr('stroke', col)
+        .delay((d) => (randomDelay * Math.random() * 1000) + (d.x * xDelay) + (d.y * yDelay))
+        .attr('r', () => 0);
+    }
+  }
+}
+
+setInterval(toggle, duration * 3);
+toggle();
```
Comment (ashb) [inline airflow/www/static/js/circles.js]: Should these be moved in to a DOM onready?

File: airflow/www/static/js/duration_chart.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    $('#cum_dur_chart').hide();
+  }
+}
+$(document).on('chartload', () => {
+  handleCheck();
+});
+
+$('#isCumulative').on('click', () => {
+  handleCheck();
+});
```
Comment (ashb) [inline airflow/www/static/js/duration_chart.js]: Couldn't we do this?

```suggestion
$(document).on('chartload', handleCheck);
$('#isCumulative').on('click', handleCheck);
```

**Review by ashb — CHANGES_REQUESTED** [review]
Few small changes.

File: airflow/www/templates/airflow/dag_code.html [hunk 5]
```diff
@@ -21,6 +21,14 @@
...
+    <meta name="demo_mode" content="{{ demo_mode }}">
+    <meta name="wrapped" content="{{ wrapped }}">
+  </head>
```
Comment (ryanahamilton) [inline airflow/www/templates/airflow/dag_code.html]: I don't _think_ there is a `head` block, but instead `head_meta`, `head_js`, or `head_css`. The `<head></head>` tags will already be provided by the the parent template.
```suggestion
{% block head_meta %}
  {{ super() }}
  <meta name="demo_mode" content="{{ demo_mode }}">
  <meta name="wrapped" content="{{ wrapped }}">
```

**Review by ryanahamilton — CHANGES_REQUESTED** [review]
Much appreciated work, thank you @bbovenzi!

File: airflow/www/templates/airflow/dag_code.html (hunk 5 above)
Comment (ashb) [reply airflow/www/templates/airflow/dag_code.html]: https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/templates/appbuilder/init.html is the template that is providing most of the "base" HTML

**Comment by bbovenzi** [comment]
@ryanahamilton @ashb Added the requested changes.

File: airflow/www/templates/airflow/dag_code.html (hunk 2 above)
Comment (ashb) [reply airflow/www/templates/airflow/dag_code.html]: Still need to remove these tags

Comment (bbovenzi) [reply airflow/www/templates/airflow/dag_code.html]: of course. my bad.

**Review by ashb — APPROVED** [review]

**Comment by ashb** [comment]
Looks good now @bbovenzi -- conflicts to resolve though.

**Review by ryanahamilton — APPROVED** [review]

**Pull request merged**